nixos/modules/services/networking/nebula.nix

```diff
@@ -84,6 +84,28 @@ in
                 description = "Whether this node is a relay.";
               };
 
+              lighthouse.serveDns = lib.mkOption {
```

Comment:
May be a good idea to move this to `lighthouse.dns.enable` so it's all under the same set of options.

Reply:
ok, respun